client/src/config.js

```diff
@@ -20,15 +21,19 @@ export const config = {
         songLinkInput: "Paste link here e.g. https://www.youtube.com/watch?v=zumC_C0C29c",
     },
     titleContent: "Youtube Converter",
-    stateKeyName: "state",
+    stateLocalStorageKey: "state",
     formats: ["mp3", "mp4", "wav"],
     defaultFormat: "mp3",
     enterKey: "Enter",
     icons: {
         delete: "☓"
     },
     songsTableHeaders: ["Name", "Format","Artist", "🕒", "Uploaded At"],
-    download_song_info_error_message: "Some error accrued, probably the server is down. Try again in a few minutes.",
+    messages: {
+        download_song_info_error_message: "Some error accrued, probably the server is down. Try again in a few minutes.",
```

Comment:
```suggestion
downloadSongInfoErrorMessage: "...
```
keep one convention - snake_case is O.K but in this entire file you use camelCase so better of keep this convention to all variables

Reply:
I didn't notice that😆